src/Components/Results/ProgramPage/NavigatorLanguages.tsx

```diff
@@ -2,7 +2,7 @@ import { FormattedMessage } from 'react-intl';
 import { Language } from '../../../Assets/languageOptions';
 import { FormattedMessageType } from '../../../Types/Questions';
 
-export const allNavigatorLanguages: Record<Language, FormattedMessageType> = {
+export const allNavigatorLanguages: Record<Language, FormattedMessageType> & Record<'ALL', FormattedMessageType> = {
```

Comment:
Is this right? I'm not a Typescript expert but wouldn't this mean it needs to satisfy both contracts?